src/web/InputWaiter.mjs

```diff
@@ -122,51 +614,90 @@ class InputWaiter {
         }
 
         document.getElementById("input-info").innerHTML = msg;
+
     }
 
+    /**
+     * Debouncer to stop functions from being executed multiple times in a
+     * short space of time
+     * https://davidwalsh.name/javascript-debounce-function
+     *
+     * @param {function} func - The function to be executed after the debounce time
+     * @param {number} wait - The time (ms) to wait before executing the function
+     * @param {array} args - Array of arguments to be passed to func
+     * @returns {function}
+     */
+    debounce(func, wait, args) {
```

Comment:
Setting this to static seems to break it. Perhaps it could be moved out of inputWaiter? (could be handy if we want a debounce anywhere else in the app)

Reply:
Agreed.

Reply:
I've moved it to `App`, and also used it for dragging the splitter and resizing the window